lib/settings/pages/debug_settings_page.dart

```diff
@@ -114,6 +121,33 @@ class DebugSettingsPage extends StatelessWidget {
               ),
             ),
           ),
+          const SliverToBoxAdapter(child: Divider(indent: 16.0, endIndent: 16.0)),
```

Comment:
Should we make this divider more like the other one we have in the appearance settings (or make that one like this one)?

https://github.com/thunder-app/thunder/blob/eade1f7a7f2c2167d31a4d04f889e357b43876fe/lib/settings/pages/appearance_settings_page.dart#L42-L50

Reply:
Done! I've matched it to be the same as the appearance page